docs/tutorials/tutorial_start_EOS-S3_HW.md

```diff
@@ -0,0 +1,217 @@
+# Getting started with QuickLogic EOS S3 Board
```

Comment:
Consider renaming file to e.g. `getting_started_eos_s3.md` so it better relates to the actual document